drivers/dma/Kconfig.stm32

```diff
@@ -27,8 +27,9 @@ config DMA_STM32_V2
 
 config DMAMUX_STM32
 	bool
+	default y
 	depends on DMA_STM32_V2
-	depends on SOC_STM32L4R5XX || SOC_SERIES_STM32WBX
+	depends on SOC_STM32L4R5XX || SOC_STM32L4R9XX || SOC_STM32L4S5XX || SOC_SERIES_STM32WBX
 	help
 	  Enable DMAMUX support on L4R/WB series SoCs.
```

Comment:
--> Enable DMAMUX support on L4S/L4R/WB series SoCs.

Reply:
I looked at the dmamux driver and from what I understand, the channel selection occurs in the function `dma_stm32_id_to_stream`. The low level driver for stm32l4 has all the relevant definitions (`LL_DMA_CHANNEL_1` to `LL_DMA_CHANNEL_7`) and has no define for `LL_DMA_CHANNEL_0`. It seems that actually `LL_DMA_CHANNEL_1` points to channel 0. In addition, I enabled both channels 0 and 13 for dmamux in my app and it worked. What do you think?